### PR TITLE
Design web view and mobile view for FaqPage

### PR DESCRIPTION
**Pull request by IRANKUND**
create web vieWhat does this PR do?
Creating Faqs page

Description of Task to be completed?
-create Faqspaje.js
-add style
-FaqstStyle.js
-create Faq route

How should this be manually tested?
-download source code run 'npm i' and run npm run start and write 'http:localhost:8080' and click in 'menu and choose Faqs'

Any background context you want to provide?
N/A

What are the relevant trello board stories?
https://trello.com/c/1DrByIvb/41-implement-faq-page-design-using-the-new-mockup-design

Screenshots (if appropriate)w and mobile view for about
faq web
faqmobile

### CODE REVIEW

**Review by maximeish — CHANGES_REQUESTED** [review]
This is good work @IRANKUND some things to fix would be to:

- move all the CSS files in shared/styles/ like we were told and if possible use emotion

- remove the newline eslint errors

**Comment by IRANKUND** [comment]
> This is good work @IRANKUND some things to fix would be to:
> 
> * move all the CSS files in shared/styles/ like we were told and if possible use emotion
> * remove the newline eslint errors

oky  am giong to work on it

**Comment by IRANKUND** [comment]
> This is good work @IRANKUND some things to fix would be to:
> 
> * move all the CSS files in shared/styles/ like we were told and if possible use emotion
> * remove the newline eslint errors
I changed it

File: src/__test__/faqspage.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    expect(wrapper).not.toBeNull(); 
+  });
+  it("switch Answers ", () => {
+    const { getByTestId } = render(<Faq faq={faq} toggleFaq={toggleFaq} />);
+    fireEvent.click(getByTestId("faqs"));
+  });
+ 
+
+
+})
```
Comment (maximeish) [inline src/__test__/faqspage.test.js]: The changes look good but there are still some newline eslint errors. Like here and in `src/shared/styles/FaqsStyle.js`
I suggest using prettier vs code extension as it automatically adds the newlines

Comment (IRANKUND) [reply src/__test__/faqspage.test.js]: oky let me do it

Comment (IRANKUND) [reply src/__test__/faqspage.test.js]: i made the  changes

**Review by maximeish — APPROVED** [review]

File: src/__test__/faqspage.test.js (hunk 1 above)
Comment (malaba6) [reply src/__test__/faqspage.test.js]: @IRANKUND the coverage dropped to much. You did not test your code. You increase the coverage to at least 92%

**Pull request merged**